vite.config.ts

```diff
@@ -4,12 +4,18 @@ import path from 'path';
 
 export default defineConfig({
   plugins: [react()],
+  resolve: {
+    alias: {
+      '@': path.resolve(__dirname, 'src'),
+    },
+  },
   css: {
     preprocessorOptions: {
       scss: {
         additionalData: `
-          @import "${path.resolve(__dirname, 'src/styles/constants.scss')}";
-          @import "${path.resolve(__dirname, 'src/styles/breakpoints.scss')}";
+          @use "@/styles/constants.scss" as *;
```

Comment:
wow :D It's the 7th combination of random approaches here. Good job that you made it work!